README.md

```diff
@@ -32,10 +35,10 @@ registration.agree_terms
 # We need to prove that we control the domain using one of the challenges method.
 authorization = client.authorize(domain: 'example.org')
 
-# For now the only challenge method supprted by the client is http-01.
+# For now the only challenge method supported by this client is http-01.
```

Comment:
We have full support for all methods implemented in Boulder, so I think we could just remove the mention.

The only missing challenge from the acme spec is proof-of-possession but I prefer not add it until a server side implementation is completed.

> I'm not sure I'd add more examples here, it's probably time for some more structured documentation than a single big example.
 
I agree, we could start linking to the wiki or create an example/ directory.